File: ios/Classes/Enums/SecureDeviceContextMethod.swift
internal enum SecureDeviceContextMethod: String {
    case isDevMode = "is_dev_mode"
    case isJailbreak = "is_jailbreak"
    case isEmulator = "is_emulator"
    case isDebugMode = "is_debug_mode"
    case status
    case contexts
    case riskLevel = "risk_level"
}

File: lib/src/enums/secure_device_context_method.dart
enum SecureDeviceContextMethod {
  isDevMode('is_dev_mode'),
  isJailbreak('is_jailbreak'),
  isEmulator('is_emulator'),
  isDebugMode('is_debug_mode'),
  status('status'),
  contexts('contexts'),
  riskLevel('risk_level');

  const SecureDeviceContextMethod(this.value);

  final String value;
}
